docs/feature-summary.md

```diff
@@ -0,0 +1,77 @@
+# Feature Summary
+
+## GitLab to Mattermost Notifications
+
+### Channel Subscriptions
+
+Notify your team of the latest updates by sending notifications from your GitLab group or repository to Mattermost channels.  When team members log in the first time to Mattermost each day, they can get a post letting them know what issues and merge requests need their attention.  They can also get a refresh of new events by clicking on *Refresh* and from every webhook configured in GitLab.
+
+You can specify which events trigger a notification. They can see:
+
+- issues - includes new and closed issues
+- merges - includes new and closed merge requests
+- pushes - includes pushes
+- issue_comments - includes new issue comments
+- merge_request_comments - include new merge-request comments
+- pipeline - include pipeline
+- tag - include tag creation
+- pull_reviews - includes merge request reviews
+- label:"<labelname>" - must include "merges" or "issues" in feature list when using a label
+- Defaults to "merges,issues,tag"
+
+
+
+![image](.gitbook/assets/image.png)
+
+
+
+### Personal Notifications: GitLab Bot
+
+Each user in Mattermost is connected with their own personal GitLab account.  Users can get a direct message in Mattermost when someone mentions them, requests their review, comments on or modifies one of their merge requests/issues, or assigns them on GitLab.
+
+
+
+### Sidebar Buttons
+
+Team members can stay up-to-date with how many reviews, unread messages, assignments and open merge requests they have by using buttons in the Mattermost sidebar.
+
+
+
+## Mattermost Commands
+
+Interact with the GitLab plugin using the `/gitlab` slash command
+
+### Subscribe to / unsubscribe from a repository
+
+Use `/gitlab subscribe owner[/repo] [features]` to subscribe a Mattermost channel to receive posts for new merge requests and / or issues, or other features (as listed above), from a GitLab repository. 
+
+Use `/gitlab unsubscribe owner/repo` to unsubscribe from it.  
+
+`/gitlab subscribe list` lists what you have subscribed to.
+
+### Connect to / disconnect from GitLab
+
+Connect your Mattermost account to your GitLab account using `/gitlab connect` and disconnect it using`/gitlab disconnect`. 
+
+`/gitlab me` displays the connected GitLab account.
+
+### Get to-do items
+
+Use `/gitlab todo` to get a list of unread messages and merge requests awaiting your review
+
+### Update Settings
+
+Use `/gitlab settings [setting] [value]` to update your settings for the plugin.  There are two settings:
+
+- To turn **personal notifications** *on* or *off*
```

Comment:
what is the full command to be used here?  Are these examples?: `/gitlab settings personal notifications off`, `/gitlab settings reminders off`

If so, we should list the settings available:
- `personal notifications` - turns off notifying you after you've personally made an update to a record
- `reminder` - Stop reminding the user about XYZ

Reply:
Aaron,

In the feature summary list there were just a handful of commands mentioned before with the last one saying just check help.  Romain suggested adding connect, which comes later in Configuration I think, but it made sense here.  When I looked at all the commands, I felt that the list was short enough that I could probably enhance each of those entries to just cover all the commands.  I don't know if I should cover all the commands clearly identifying the options, or just leave that to the /gitlab help.  

For example, with the gitlab settings I used bold and then italics for the [setting] and [value] options.  Perhaps too subtle?  

So, I could go back to not detailing out the syntax or completely detailing them out.  Let me know what you think.

Reply:
Also, on another topic, I was trying to test this gitbook and the pages were not rendering in the Table of Contents sequence as they do with the Jira gitbook.  Is there something I need to do in order to get that working correctly?

Reply:
FYI @aaronrothschild on above